src/public/components/text-input.js

```diff
@@ -0,0 +1,7 @@
+import K, * as U from "karet.util"
+import React     from "karet"
+
+export const TextInput = ( { value } ) =>
+  <input type="text"
+         value={ value }
+         onChange={ U.getProps( { value } ) }/>
```

Comment:
Would it make sense to have most components accept `mount` and `...props`?

Also, do you have an example that shows how to work with the `ref` translated from a `mount`? I like to set focus on inputs intelligently / automatically (so the user doesn't have to tab through or use their mouse to put focus where you're pretty sure they'd want to have it after some action). For instance, in `QuerystringParams`, I think it would make sense to set focus to the 'key' input of a newly added key/value pair after the user clicks the 'Add' button.